### ISSUE TITLE
SMS Sharing on Share Buttons

### ISSUE THREAD

**Issue by mattymattyonga** [elementor/elementor#7436]
## Prerequisites

- [x] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [x] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**
Hi, Sorry not familiar with Github, hope this is all in order. Currently I use Addtoany sharing buttons but I would like to replace that with the built in Sharing buttons. One option that's missing that we need is SMS sharing.

**Describe the solution you'd like**
SMS sharing would be an option amongst the sharing buttons

**Describe alternatives you've considered**
Using seperate plugin

**Additional context**
None needed

Thanks for Elementor by the way, it's a game changer for me!

**Comment by drawcard** [comment]
I have a feeling this would require a SMS gateway from a mobile network provider, or 3rd party service of some kind on top of Elementor.

You'd probably be better off using a third party solution and integrating their button into Elementor (or using the [Forms API](https://developers.elementor.com/forms-api/) to integrate into a webform)

**Comment by drawcard** [comment]
Also have a look at hooking up Elementor + SMS with Zapier:
https://docs.elementor.com/article/263-zapier
https://zapier.com/apps/sms/integrations

**Comment by mattymattyonga** [comment]
Sorry I should have been more specific, what I was meaning was the sharing button would initiate a message that shares the link of the page in a text message - as in, it opens a the text message program on the phone or computer and have the body prefilled with the link.

**Comment by mattymattyonga** [comment]
For instance, you can see the plugin I use currently has this as one of the sharing options: https://theincrediblejourney.tv/shows/captain-cook-master-of-the-seven-seas/
buttons below the video.

**Comment by drawcard** [comment]
Oh right, I thought you meant you needed Elementor to send an SMS to someone, rather than just have a button that launches an SMS message app.

**Comment by shilo-ey** [comment]
Hi @mattymattyonga 
You can do this with any link > Dynamic > [Contact URL > SMS](https://docs.elementor.com/article/448-dynamic-contact-url)

Unfortunately, due to different encoding in different devices, we have issues with adding the message itself. but we will take a look to see if we can add this option.

Thanks.

**Comment by therarecommoner** [comment]
I think what the OP is suggesting is not a way to contact a predetermined phone number as is the functionality defined in link > Dynamic > Contact URL > SMS, but to populate an empty SMS where a user would then apply a phone number to share content. 

Would be really great to see this added.

**Comment by mwittmann** [comment]
Yes, would love to have an native SMS share button option for mobile!

**Comment by ianlee** [comment]
If this is not possible, Can we get Copy to clipboard button instead?

**Comment by tjuays** [comment]
I am also desperately looking for this functionality and it would be a simple button to add to the existing sharing options. Elementor just needs to create the string something like sms:&body="Check out this great website" [page_url]

If I could get an update on the possibility of this being implemented in the near future I would appreciate it.
Thanks

**Comment by muzKore** [comment]
This. would be fantastic to have as an option. I managed to get this working by adding an icon block next to the share buttons block and styled it accordingly.  As I needed the URL to be dynamic, I set the URL of the icon block to point to a shortcode and then got the URL of the current post/page and then generated the link and passed it back to be placed in the SMS button link.

**Comment by joehdevwp** [comment]
+1 for this. A button to share the current post/url or copy to clipboard as a less good alternative
